src/components/Create/components/RewardsList/AddEditRewardModal.tsx

```diff
@@ -236,19 +282,108 @@ export const AddEditRewardModal = ({
+          activeKey={advancedOptionsOpen ? ['0'] : []}
+          onChange={onCollapseChanged}
         >
-          <PrefixedInput prefix="https://" />
-        </Form.Item>
+          <CreateCollapse.Panel
+            header={<OptionalHeader header={t`Advanced options`} />}
+            key={0}
+            hideDivider
+          >
+            <Form.Item
+              extra={
+                <span className="text-sm">
+                  <Trans>Set aside a percentage of freshly minted NFTs.</Trans>
+                </span>
+              }
+            >
+              <JuiceSwitch
+                value={isReservingNfts}
+                onChange={setIsReservingNfts}
+                label={
+                  <span className="flex items-center gap-2">
+                    <Trans>Reserve NFTs</Trans>
+                    <Tooltip
+                      className="cursor-help text-grey-500 dark:text-grey-300"
+                      title={t`If Reserved NFTs are set, the first reserved NFT from the tier will be distributable once the tier receives its first mint.`}
+                    >
+                      <QuestionCircleOutlined />
+                    </Tooltip>
+                  </span>
+                }
+              />
+            </Form.Item>
+            {isReservingNfts && (
+              <>
+                <span className="mb-4 flex flex-col gap-2 whitespace-nowrap md:flex-row md:items-center">
+                  <Trans>Reserve 1 NFT of every</Trans>
+                  <div className="max-w-[78px]">
+                    <Form.Item
+                      className="mb-0"
+                      name="nftReservedRate"
+                      rules={[
+                        inputMustExistRule({ label: t`Reserved rate` }),
+                        inputIsIntegerRule({
+                          stringOkay: true,
+                        }),
+                      ]}
+                    >
+                      <FormattedNumberInput min={2} placeholder="0" />
+                    </Form.Item>
+                  </div>{' '}
+                  <Trans>NFTs minted for:</Trans>
+                </span>
+                <Form.Item
+                  className="mb-8"
+                  name="beneficiary"
+                  label={t`Ethereum wallet address`}
+                  extra={t`Reserved NFTs will be sent to this address once minted.`}
+                  rules={[
+                    inputMustExistRule({ label: t`Wallet address` }),
+                    inputMustBeEthAddressRule({ label: t`Wallet address` }),
+                  ]}
+                >
+                  <EthAddressInput />
+                </Form.Item>
+              </>
+            )}
+            <Form.Item
+              name="votingWeight"
+              label={t`Voting weight`}
+              tooltip={t`Give this NFT a voting weight to be used for on-chain governance. The number you set is only used in relation to other NFTs in this collection.`}
```

Comment:
Wdyt about switching the `extra` and `tooltip` texts here? I feel the tooltip is more essential, and having the current `extra` text exposed adds a lot of (in many cases, unnecessary) text to the modal.

Reply:
![image](https://user-images.githubusercontent.com/104132113/214976320-19497f8f-b6e5-40d8-8617-a2088f3042fb.png)
Switching has about the same amount of text. I am basing off @strath-m initial design here:
https://www.figma.com/file/EN7CZTekcSo3sHyaGt6cC0/JBX---Project-Creation-Flow---High-Fidelity---Updated-Jan-20223?node-id=2466%3A156448&t=woX39OFlKH1vNxrF-0

The copy changed a bit in discord

Reply:
Have switched them around!